src/API/Middlewares/HttpExceptionsMiddleware.test.ts

```diff
@@ -0,0 +1,97 @@
+import { IllegalArgumentException, NotFoundException } from '@po.et/poet-js'
+import { describe } from 'riteway'
+import { spy } from 'sinon'
+import { getCodeError, HttpExceptionsMiddleware, executeLogger } from './HttpExceptionsMiddleware'
+
+describe('HttpExceptionsMiddleware middleware', async (should: any) => {
+  const { assert } = should('')
+  {
+    const ctx = {}
+    const next = spy()
+    await HttpExceptionsMiddleware(ctx, next)
+
+    assert({
+      given: 'no errors',
+      should: 'call next once',
+      actual: next.calledOnce,
```

Comment:
Dunno how this works, but doesn't `next` get called every time no matter what?

```ts
  try {
    await next()
  } catch ...
```

Maybe we should test that `context.body` and `context.status` were not mutated by `HttpExceptionsMiddleware`?

Reply:
well, just calling the spy you are securing the `next` resolve with success. But I can add the check the immutability the `ctx` also

Reply:
done!

Reply:
Thanks, but my point is that `next.calledOnce === true` doesn't mean there are _no errors_, which is what we're saying in the `given` clause.

`.next` gets always *always*, errors or no errors, so this unit test doesn't really test anything.

I wrote an small example of this: https://github.com/poetapp/node/pull/375

Reply:
@wzalazar what about this one?

Reply:
I changed the given to `given: 'no errors or errors'`, but I'm not sure with it, what do you say?

Reply:
Even more confusing 😛 

If we remove the test, does the coverage drop?

Reply:
yes, maybe is more confusing... ahahahah

yes, I can remove this test without suffering drop coverage

Reply:
Awesome 🚀